app/templates/post.hbs

```diff
@@ -1,35 +1,32 @@
-
 {{#let @model.post as |post|}}
 <article class="post-content {{unless post.image 'no-image'}}">
-
-    <header class="post-content-header">
-        <h1 class="post-content-title">{{post.title}}</h1>
-    </header>
-
+    {{!-- 
     {{#if post.image}}
```

Comment:
Still need to implement this

Reply:
Actually I was going to mention that we don't currently support "cover images" for posts so we can probably just delete it for now 👍

Reply:
Awesome. Will do. Thanks